ReactWindows/ReactNative.Shared/Bridge/Queue/MessageQueueThread.cs

```diff
@@ -281,7 +281,10 @@ public AnyBackgroundMessageQueueThread(string name, Action<Exception> handler)
                 _taskFactory = new TaskFactory(_taskScheduler);
             }
 
+            [System.Diagnostics.CodeAnalysis.SuppressMessage("AsyncUsage.CSharp.Reliability", "AvoidAsyncVoid", Justification = "Reviewed.")]
```

Comment:
`#pragma warning disable/enable` is to suppress violations for false positives, and `SuppressMessage` is an optional code that specifies the analysis tool, identifier and so on.